src/models/request/registerRequest.ts

```diff
@@ -1,6 +1,7 @@
 import { KeysRequest } from './keysRequest';
 
 import { KdfType } from '../../enums/kdfType';
+import { ReferenceEventData } from '../domain/referenceEventData';
```

Comment:
can move this up with the other request model

Reply:
Done